jib-cli/CHANGELOG.md

```diff
@@ -6,11 +6,15 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 ### Changed
-- Upgraded Google HTTP libraries to 1.42.2 ([#3745](https://github.com/GoogleContainerTools/jib/pull/3745))
-- Re-synchronized jackson dependencies with BOM to use latest versions ([#3768](https://github.com/GoogleContainerTools/jib/pull/3768))
 
 ### Fixed
 
+## 0.12.0
+
+### Changed
+- Upgraded Google HTTP libraries to 1.42.2 ([#3745](https://github.com/GoogleContainerTools/jib/pull/3745))
+- Re-synchronized jackson dependencies with BOM to use latest versions ([#3768](https://github.com/GoogleContainerTools/jib/pull/3768))
+
```

Comment:
How about we hold off on the jib CLI doc updates until the release job succeeds?

Reply:
Oh good catch, I accidentally committed these too! Will remove.